docs/index.md

```diff
@@ -56,10 +56,25 @@ Currently, Pulse provides a push-based integration with your workflow using one
 
 Share the Pulse dashboards with your teammates to bring more visibility to the metrics.
 
-To do this, copy the link on the **Organization members** page and share it with your teammates. A convenient way to do this is to share and pin the link on your team's chat.
+The most convenient way of ensuring that all the members of your company have access to the Pulse dashboards is by enabling the feature **Joining via email domain** on the **Organization members** page. When this option is enabled, anyone who signs up to Pulse using an email address from your company is automatically added to your Pulse organization.
 
-![Inviting teammates to Pulse](images/inviting-members.png)
+Alternatively, copy the invitation link from the **Organization members** page and share the link with specific people. A convenient way to do this is to share and pin the link on your team's chat. The link is valid for 3 months, but you can refresh it at any moment.
 
-By doing this, the people you invite will have access to all dashboards immediately after signing up without needing to perform any integration.
+![Adding organization members](images/joining-email-domain.png)
 
-The link is valid for 3 months, but you can refresh it at any moment.
+Newcomers will have access to all dashboards immediately after signing up without needing to perform any integration.
+
+!!! note "About joining via email domain"
```

Comment:
Not sure if it's relevant to mention that turning off this feature won't remove from the org users that joined when the feature was active.

Reply:
It's a good point, and I've added it to the list of "further considerations" about the feature.